scripts/_helpers.py

```diff
@@ -25,11 +27,14 @@ def configure_logging(snakemake, skip_handlers=False):
     kwargs.setdefault("level", "INFO")
 
     if skip_handlers is False:
+        fallback_path = Path(__file__).parent.parent.joinpath('logs', f"{snakemake.rule}.log")
+        logfile = snakemake.log.get('python', snakemake.log[0] if snakemake.log
+                                    else fallback_path)
```

Comment:
@euronion you agree with that? for debugging in scripts folder, it is necessary to set the path independent of the cwd

Reply:
No, I can not agree with this change.

* First: See the comment above, the second `.parent` in your code has not effect.
* Second: I do not fully understand your comment, as it still depending on the `cwd` (but using it as a relative anchor).
* Also: If you really use the parents parent `..`, then it breaks the original behaviour for the case of neither `snakemake.log.python`  nor `snakemake.log` being defined. Instead of `logs/xyz.log` it would use `../logs/xyz.log` as a path.

If you are doing debugging and encounter this problem, you could still `skip_handlers` and add them manually to the `logging` root logger.

Reply:
I tested the current version, for `snakemake.log.python` and `snakemake.log` being undefined, and it works for both `Snakemake` and `mock_snakemake`. I don't care about the specific implementation, both  

* `Path(__file__).resolve().parent.parent.joinpath('logs', f"{snakemake.rule}.log")`
* `Path(__file__).parent.joinpath('..', 'logs', f"{snakemake.rule}.log")`

I'd go for the second as it is a bit shorter